tests/dummy/app/pods/toggle/template.hbs

```diff
@@ -0,0 +1,104 @@
+<div class='dummy-notification-container'>
+  {{#each notifications as |notification|}}
+    {{notification-message notification=notification}}
+  {{/each}}
+</div>
+
+<div class='dummy-body button'>
+    <div class='section-title'>Toggle button</div>
+    <hr>
+    <div class='section'>
+        <div class='example'>
+            <div class='title'>action syntax</div>
+            <div class='demo'>
+              {{! BEGIN-SNIPPET button-primary-small }}
+              {{frost-toggle-button
+              onToggle=(action 'toggleHandler')}}
+              {{! END-SNIPPET }}
+            </div>
+            <div class='snippet'>
+              {{code-snippet name='button-primary-small.hbs'}}
+            </div>
+        </div>
+
+        <div class='example'>
+            <div class='title'>mut syntax</div>
+            <div class='demo'>
+              {{! BEGIN-SNIPPET button-primary-medium }}
+              {{frost-toggle-button
+              toggled=isButtonToggled
+              onToggle=(mut isButtonToggled)
+              }}
+              {{! END-SNIPPET }}
+            </div>
+            <div class='snippet'>
+              {{code-snippet name='button-primary-medium.hbs'}}
+            </div>
+        </div>
+
+
+        <div class='example'>
+            <div class='title'>label</div>
+            <div class='demo'>
+              {{! BEGIN-SNIPPET button-primary-large }}
+              {{frost-toggle-button
+              onLabel='6500'
+              offLabel='5400'
+              onToggle=(action 'toggleHandler')
+              }}
+              {{! END-SNIPPET }}
+            </div>
+            <div class='snippet'>
+              {{code-snippet name='button-primary-large.hbs'}}
+            </div>
+        </div>
+
+        <div class='example'>
+            <div class='title'>value</div>
+            <div class='demo'>
+              {{! BEGIN-SNIPPET button-primary-disabled }}
+              {{frost-toggle-button
+              onValue='on Value'
```

Comment:
I get why you called this onValue - but it's a bit confusing given our event naming conventions `on*` so maybe we can call it trueValue/falseValue or even true/false?